src/common/version.cc

```diff
@@ -23,15 +23,14 @@
 #define _STR(x) #x
 #define STRINGIFY(x) _STR(x)
 
-// Keep ver.c_str() available
-static std::string ver;
-
-const char *ceph_version_to_str(CephContext *cct)
+const char *ceph_version_to_str()
 {
-  if (cct) ver = cct->_conf.get_val<std::string>("debug_version_for_testing");
-  if (ver.size() > 0)
-    return ver.c_str();
-  return CEPH_GIT_NICE_VER;
+  char* debug_version_for_testing = getenv("debug_version_for_testing");
```

Comment:
There is a (*theoretical*, I believe) difference in comparison to the previous implementation: clash of environment variable names. What if another project would like to use the same one? Maybe we should prefix it like `ceph_debug_version_for_testing`?

Reply:
@rzarzynski fixed and repushed.